keyboards/0_sixty/rules.mk

```diff
@@ -19,3 +19,5 @@ BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
+
+DEFAULT_FOLDER = 0_sixty/base
```

Comment:
Honestly, no need to use "base".  In fact, I'd say that doing so makes things more complicated.  You can compile just `0_sixty` and get the base, and then `0_sixty/underglow` if you need. 

You can take a look at the ergodox ez, and the glow and shine revisions for an example of this.

Reply:
This looks like a better way of handling it. THank